packages/marshal/docs/smallcaps-cheatsheet.md

```diff
@@ -0,0 +1,29 @@
+# Smallcaps Cheatsheet
+
+An example-based summary of the Smallcaps encoding
+
+| Passable value  | OCapn name    | JS example          | JSON encoding        |
+| ----------------|---------------|---------------------|----------------------|
+| bigint          | SignedInteger | `7n`<br>`-7n`       | `"+7"`<br>`"-7"`     |
+| manifest constant | Undefined<br><br>Float64<br><br><br> | `undefined`<br>`Infinity`<br>`-Infinity`<br>`NaN`<br>`-0` | `"#undefined"`<br>`"#Infinity"`<br>`"#-Infinity"`<br>`"#NaN"`<br>`"#-0"` // unimplemented |
+| passable symbol | Symbol        | `Symbol.for('foo')`<br>`Symbol.asyncIterator` | `"%foo"`<br>`"%@@asyncIterator"` |
```

Comment:
It might be good to also illustrate how registered symbols get escaped so they do not overlap well-known symbols.

Reply:
Awaiting feedback from @cwebber , but even for JS this seems too detailed for a cheetsheet. OTOH, the whole thing is so close to being correct and complete that it might be worth the expansion, and then relabeling the document from "cheetsheet" to something else.